extensions/cornerstone-dicom-sr/src/utils/addSRAnnotation.ts

```diff
@@ -25,7 +28,12 @@ export default function addSRAnnotation(measurement, imageId, frameNumber) {
   }
 
   if (valueType === 'SCOORD3D') {
```

Comment:
This is really quite ugly - the probe adapter should just be updated to handle the SRSCOORD3DPoint directly as a registered adapter, and handle the conversion internally.  That is NOT part of this PR, but could you create an issue in OHIF to fix that?

Reply:
There's an IDC demand for 3D coordinates. They use annotations that are not generated by CS3D but rather other dicom-compatible software.

This SRSCOORD3DPoint assignment was part of this effort and this particular line of code was added by another developer. However they've recently put me in charge of this demand, so I'll be looking into improving this part really soon.

Having said that we should discuss a way of making OHIF able to read these reports that are indeed dicom compatible but that were not generated by CS3D, as it now relies entirely on the SR annotation having a reference to a CS3D TrackingIdentifier.